apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,129 @@
+
+### The cost of coordination
+
+The second major blocker to mainstream development and adoption of simulation is one of coordination. Simulations have historically required handoff between:
+
+1.  data engineers and analysts who make the data available which is required to instantiate 'agents' in a simulation, and hydrate them with 'properties' that describe them accurately;
+2.  programmers who're able to encode subject matter expertise _in code_ that provides 'behavioral logic' to the agents, and environment within which they reside;
+3.  domain experts, who understand how the target system functions, and can actually provide the subject matter expertise and model design required;
+4.  data scientists, who run the actual experiments required to derive real insight from probabilistic models; and
+5.  decision makers, who are understandably deeply skeptical of whiz-bang simulation models that promise to predict the future. It's "shit in, shit out" with these things, and with so many steps and people involved, by the time a model has been built and experiments run it's entirely fair (and a good idea) to have questions about model integrity.
+
+The number of different stakeholders, and amount of multi-party communication involved in a single iteration loop required to make simulation models accurate _and useful_ often proved killer to getting simulations off the ground. The average model has historically required many such iterations.
+
+Now, with the picture around data structuring and availability much improved by foundation models (as outlined above), we believe such models' further potential lies in **enabling domain experts to bypass programmers and iterate on simulation logic directly.** Behaviors within simulations are generally relatively simple, standalone things. Natural language behavior developement seems eminently achievable using current-generation (e.g. GPT-4 or fine-tuned GPT-3.5) models alone.
+
+### Structure and simulation
+
+The role of structured data in all of this is quite simple.
+
+**Predictive simulations of real-world things are extremely easy to get wrong.** Many interesting systems that users typically want to simulate end up being "complex systems" in some respect, demonstrating some degree of nonlinearity due to emergent or adaptive behaviors. This is both because (a) any system involving a human is inherently complex, and because (b) "closed systems" (the only ones we can fully model without doubt, where all parameters and environmental variables are known) quite simply do not exist in the real world. Simulations are simplifications of the real world: downsized, abstracted, or analogical.
```

Comment:
Could this para directly address structured data's role? The following two do so helpfully. or is this a set up for the next two? If the latter might be worth formatting it differently